dockerfiles/hpcc/values.schema.json

```diff
@@ -246,6 +261,44 @@
         },
         "image": {
           "$ref": "#/definitions/image"
+        },
+        "logging": {
+          "type": "object"
+        }
+      }
+    },
+	"logging": {
+      "type": "object",
+      "properties": {
+        "fields": {
+          "type": "string",
```

Comment:
Would this be more natural in a HELM file as a list rather than a string?  Same for audiences/classes.  (I don't know the answer.)

Reply:
Yes, we discussed making these a list, it was decided instead to carry over the legacy format which utilizes pre-stablished logic,and is more flexible, allowing users to build up fields(with the + delimter) or build down with the - delimiter